src/content/lodash/array/toPairs/vanilla.js

```diff
@@ -0,0 +1,9 @@
+function Foo() {
+  this.a = 1
+  this.b = 2
+}
+
+Foo.prototype.c = 3
+
+module.exports = Object.entries(new Foo())
```

Comment:
`Object.entries` can be used starting from node v6, I don't think it's worth adding pre-`Object.entries` implementation